minio/error.py

```diff
@@ -204,8 +204,9 @@ def _set_error_response_with_body(self, bucket_name=None,
             raise ValueError('response data has no body.')
         try:
             root = cElementTree.fromstring(self._response.data)
-        except Exception as e:
-            raise InvalidXMLError('"Error" XML is not parsable.')
+        except _ETREE_EXCEPTIONS as error:
```

Comment:
catching Exception makes us to handle all Exception hierarchy which is bad.  Look at https://docs.python.org/2/library/exceptions.html#exception-hierarchy